{{cookiecutter.project_slug}}/.gitignore

```
@@ -337,3 +337,8 @@ MailHog
 {%- if cookiecutter.use_docker == 'y' %}
 .ipython/
 {%- endif %}
+
+{%- if cookiecutter.js_task_runner == 'Gulp' %}
+project.css
+project.min.css
+{% endif %}
```

We probably should also add a few more (see #2857):  

```suggestion
{%- if cookiecutter.js_task_runner == 'Gulp' %}
project.css
project.min.css
vendors.js
*.min.js
{%- endif %}
```